lib/tracecontext/tracestate.js

```diff
@@ -16,7 +23,7 @@
  *   const newHeader = tracestate.toW3cString()
  */
 class TraceState {
-  constructor (sourceBuffer, listMemberNamespace = 'es', defaultValues = {}, logger) {
+  constructor (sourceBuffer, listMemberNamespace = 'es', defaultValues = {}) {
```

Comment:
Alan, that `logger` arg is never used, AFAICT. Let me know if you think it should still be a feature.